server/test/test_api.py

```diff
@@ -147,6 +147,34 @@ def test_filter_put_annotations_obs(self):
         self.assertEqual(len(result_data["data"][0]), 3)
         self.assertEqual(len(result_data["data"]), 15)
 
+    def test_diff_exp(self):
+        endpoint = "diffexp/obs"
+        url = f"{URL_BASE}{endpoint}"
+        params = {
+            "mode": "topN",
+            "set1": {
+                "filter": {
+                    "obs": {"annotation_value": [
+                        {"name": "louvain", "values": ["NK cells"]}
+                    ]
+                    }
+                }
+            },
+            "set2": {
```

Comment:
super minor point - the current f/e usage is filtering by index, not annotation value.   Might want to add a test for that?

Reply:
got it